src/Components/FollowButton/FollowArtistButton.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react"
+import {
+  commitMutation,
+  createFragmentContainer,
+  graphql,
+  RelayProp,
+} from "react-relay"
+import { FollowButton } from "./Button"
+import * as Artsy from "../Artsy"
+import { FollowArtistButton_artist } from "../../__generated__/FollowArtistButton_artist.graphql"
+
+interface Props
+  extends RelayProps,
+    React.HTMLProps<FollowArtistButton>,
+    Artsy.ContextProps {
+  relay?: RelayProp
+}
+
+interface RelayProps {
+  artist?: FollowArtistButton_artist
+}
```

Comment:
Btw, this use of a separate `RelayProps` interface is a legacy pattern we used for a tool we previously had that would in-place update an interface by that name with the fields and types that were being queried. It’s not necessary to keep using that pattern, so your interface could probably just look like:

```ts
interface Props extends React.HTMLProps<FollowArtistButton>, Artsy.ContextProps {
  relay: RelayProp
  artist: FollowArtistButton_artist
}
```

(Note that I also removed the optionality from the `relay` and `artist` props, because I assume these will always be present, but that may well be an incorrect assumption.)